getting-started/vs-integration/convert-project-wizard.md

```diff
@@ -57,18 +57,28 @@ Alternatively, instead of the context menu, you can use the VS **Extensions** me
 
 ![Start the Convert Project Wizard](images/vs-ext-convert-project-entry.png)
 
-## Features
+## Converting the Project Thought the Wizard
 
-The wizard provides you with the following options:
+Clicking the Convert to Telerik Application button opens the wizard and you will go through the following steps:
+
+### Step 1: Select Initial Project Configuration
+
+In this screen, you can choose:
```

Comment:
This sentence does not correspond to the items below. For example, you get "you can choose ... A version selector". Consider something like "This screen provides the ability to:" and then start with verbs:

* Download ...
* Select a version ...
* Configure additional project settings like ....